lib/resources/submissions.js

```diff
@@ -110,6 +110,7 @@ module.exports = (service, endpoint) => {
                     return SubmissionAttachments.upsert(extant, files);
                   }
                   // NEW SUBMISSION REQUEST: create a new submission and attachments.
+                  if(!draft && !form.acceptsSubmissions()) throw Problem.user.notAcceptingSubmissions();
```

Comment:
@matthew-white we talked about linting recently. I'm surprised there isn't something to catch the lack of space after `if`?

Reply:
https://github.com/getodk/central-backend/issues/505 maybe?  Although there isn't currently a rule for spaces after `if`.

I'll get this fixed.

Reply:
It does seem like it'd be helpful to add linting to CircleCI! @alxndrsn, feel free to make that change once you get the chance.